recipes/default.rb

```diff
@@ -1,3 +1,66 @@
+# Install PHP 7.1 from PPA
+#
+apt_repository 'php7.1' do
+  distribution node['lsb']['codename']
+  uri 'ppa:ondrej/php'
+  only_if do
+    node['php']['version'] == '7.1'
+  end
+end
+
+# Toggle attributes based on php version
+node.default['php'].tap do |php|
+  case node['php']['version']
+    when '7.1'
+      php['checksum']            = '4124a3532e2ae0e1f14b8f4f73114f56499c24ce21bac58fe1d760ed94fa5ce0'
+      php['conf_dir']            = '/etc/php/7.1/cli'
+      php['src_deps']            = %w(libbz2-dev libc-client2007e-dev libcurl4-gnutls-dev libfreetype6-dev libgmp3-dev libjpeg62-dev libkrb5-dev libmcrypt-dev libpng12-dev libssl-dev pkg-config)
+      php['packages'].concat(%w(php7.1-cgi php7.1 php7.1-dev php7.1-cli php-pear))
```

Comment:
Spoke to him directly, this does something different than what he thought. `.concat` on an array merges it with what's being passed to it VS the string concatenation which he thought was what was applied here.